wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -998,7 +998,7 @@ derivation, it is unlikely that the same addresses will be generated on two
 different node instances. However, some API users may wish to preserve unused
 addresses between different instances of the wallet backend.
 
-To enable this, the wallet backend provides an endpoint ([`POST /api/v1/wallets/{{walletId}}/accounts/{{accountId}/addresses`](#tag/Addresses%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts~1{accountId}~1addresses%2Fpost))
+To enable this, the wallet backend provides an endpoint ([`POST /api/v1/wallets/{{walletId}}/accounts/{{accountId}/addresses`](#tag/Addresses%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1addresses%2Fpost))
```

Comment:
`/accounts/{{accountId}` needs to be removed.

Reply:
Also fixed. Thanks for catching this :+1: